src/main/java/com/example/usermanagement/service/UserService.java

```diff
@@ -28,41 +30,121 @@ public class UserService {
 
     private final UserMapper userMapper;
     private final UserRepository userRepository;
-    private final PasswordEncoder passwordEncoder;
     private final UserValidation userExisting;
-    private final JwtService jwtService;
-    private final AuthenticationManager authenticationManager;
+    private final WebClient webClient;
 
-    public UserService(UserMapper userMapper, UserRepository userRepository, PasswordEncoder passwordEncoder, UserValidation userExisting, JwtService jwtService, AuthenticationManager authenticationManager) {
+    @Value("${spring.security.oauth2.client.provider.keycloak.issuer-uri}")
```

Comment:
we already have a keycloakadapterconfig class, then why do we need these fileds here?

Reply:
Configured other props. and now using this class only.